master/static/migrations/20210922170815_update-model-registry-get-apis.up.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE public.modelsv2 (
+	id integer NOT NULL,
+	name character varying UNIQUE NOT NULL,
+	description character varying,
+	metadata jsonb,
+	creation_time timestamp with time zone NOT NULL,
+	last_updated_time timestamp with time zone,
+    num_versions integer DEFAULT 0 NOT NULL,
+    labels text[],
+    readme text,
+    username text,
+    archived boolean DEFAULT false NOT NULL,
+
+	CONSTRAINT modelsv2_pkey PRIMARY KEY (id)
+);
+
+ALTER TABLE public.modelsv2 ALTER COLUMN id ADD GENERATED BY DEFAULT AS IDENTITY (
+    SEQUENCE NAME public.models_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE public.model_versionsv2 (
+	version integer NOT NULL,
+	model integer NOT NULL REFERENCES public.modelsv2(id),
```

Comment:
TODO: Yeah let's look into why the FOREIGN KEY constraint at the end of the statement wasn't working for you.